Makefile

```diff
@@ -54,14 +54,14 @@ SERVER_TESTS = $(wildcard server-tests/*.t)
 
 
 default:
-	echo Nothing to do--no compilation steps.
+        echo Nothing to do--no compilation steps.
```

Comment:
Have these tabs been converted to spaces?  For most files that's okay but in a Makefile they need to be tabs.  (Same for the rest of the diffs listed for Makefile.)